lib/jekyll-seo-tag.rb

```diff
@@ -34,7 +27,7 @@ def payload
     def info
       {
         :registers => context.registers,
-        :filters   => [Jekyll::Filters]
+        :filters   => [Jekyll::Filters, Jekyll::Seo::Filters]
```

Comment:
I am trying to keep the filter only available for use by the plugin, rather than using `Liquid::Template.register_filter` to make it available everywhere.

Is there a better way to go about this?